sopel/tools/jobs.py

```diff
@@ -48,53 +55,61 @@ def peek(self):
 
 
 class JobScheduler(threading.Thread):
-
     """Calls jobs assigned to it in steady intervals.
 
     JobScheduler is a thread that keeps track of Jobs and calls them every
-    X seconds, where X is a property of the Job. It maintains jobs in a
-    priority queue, where the next job to be called is always the first
-    item.
-    Thread safety is maintained with a mutex that is released during long
-    operations, so methods add_job and clear_jobs can be safely called from
-    the main thread.
+    X seconds, where X is a property of the Job.
 
-    """
-
-    min_reaction_time = 30.0  # seconds
-    """How often should scheduler checks for changes in the job list."""
+    Thread safety is ensured with an internal mutext.
```

Comment:
```suggestion
    Thread safety is ensured with an internal mutex.
```

Reply:
You can clearly see that I wrote that on the train... x)